File: Assets/Project/SpaceEngine/Code/Tools/Wireframe.js
var state = false;

function Update() {
    if (Input.GetKeyDown("f1"))
        state = !state;
}

function OnPreRender() {
    if (state)
        GL.wireframe = true;
}

function OnPostRender() {
    if (state)
        GL.wireframe = false;
}

File: Assets/Project/SpaceEngine/Code/Utilities/Wireframe.cs
using UnityEngine;

public class Wireframe : MonoBehaviour
{
    public bool Enabled = false;

    private void LateUpdate()
    {
        if (Input.GetKeyDown(KeyCode.Slash))
            Enabled = !Enabled;
    }

    private void OnPreRender()
    {
        if (Enabled)
            GL.wireframe = true;
    }

    private void OnPostRender()
    {
        if (Enabled)
            GL.wireframe = false;
    }
}